src/components/scenes/TransactionListScene.tsx

```diff
@@ -230,9 +234,10 @@ function TransactionListComponent(props: Props) {
   })
 
   return (
-    <NotificationSceneWrapper navigation={navigation} hasTabs scroll>
+    <NotificationSceneWrapper navigation={navigation} hasTabs>
```

Comment:
Was removing the `scroll` intentional? Not a problem if so - just double-checking.

Reply:
It was. We should have the FlashList be the scroll